zb-property.js

```diff
@@ -237,6 +278,37 @@ class ZigbeeProperty extends Property {
     ];
   }
 
+  /**
+   * @method setColorValue
+   *
+   * Convert the 'color' property value (an RGB hex string) into hue
+   * and saturation values.
+   */
+  setColorValue(propertyValue) {
+    let color = new Color(propertyValue);
+    let hsv = color.hsv().color;
+    let hue = hsv[0];     // 0-359
+    let sat = hsv[1];     // 0-100
+    let level = hsv[2];  // 0-100
+
+    let levelProperty = this.device.findProperty('_level');
+    if (levelProperty) {
+      this.device.sendZclFrameWaitExplicitRx(
+        levelProperty,
+        levelProperty.valueToZclData(level));
+    }
+
+    return [{
+        frameCntl: {frameType: 1},
+        cmd: 'moveToHueAndSaturation',
+        payload: [Math.round(hue/360 * 254),
```

Comment:
Should these be 255?

Reply:
No - 254 is correct. 255 is a reserved value for most Zigbee attributes.